docs/tutorials/RunWithoutAutoware.md

```diff
@@ -0,0 +1,180 @@
+# Running `scenario_simulator_v2` without Autoware
+
+This tutorial describes all steps necessary to run `scenario_simulator_v2` without Autoware.
+
+## Prerequisites
+
+This instruction assumes operating system: Ubuntu 20.04 LTS Desktop. It can be downloaded [here](https://ubuntu.com/download/desktop/thank-you?version=20.04.4&architecture=amd64)
+
+## How to build
+
+### Install and setup ROS2 environment
+
+Instruction below is based on [official ROS2 instruction](https://docs.ros.org/en/galactic/Installation/Ubuntu-Install-Debians.html)
+
+Check locale information
+```
+locale
+```
+
+It is important that output contains UTF-8 formatting. Might be similar to text below:
+```
+LANG=en_US.UTF-8
+LANGUAGE=
+LC_CTYPE="en_US.UTF-8"
+LC_NUMERIC=pl_PL.UTF-8
+LC_TIME=pl_PL.UTF-8
+LC_COLLATE="en_US.UTF-8"
+LC_MONETARY=pl_PL.UTF-8
+LC_MESSAGES="en_US.UTF-8"
+LC_PAPER=pl_PL.UTF-8
+LC_NAME=pl_PL.UTF-8
+LC_ADDRESS=pl_PL.UTF-8
+LC_TELEPHONE=pl_PL.UTF-8
+LC_MEASUREMENT=pl_PL.UTF-8
+LC_IDENTIFICATION=pl_PL.UTF-8
+LC_ALL=
+```
+
+If it's not execute:
+
+```
+sudo apt-get update && sudo apt-get install -y locales
+sudo locale-gen en_US en_US.UTF-8
+sudo update-locale LC_ALL=en_US.UTF-8 LANG=en_US.UTF-8
+LANG=en_US.UTF-8
+```
+
+Verify setting again:
+```
+locale
+```
+
+Add ros repository for apt to use
+```
+sudo apt update && sudo apt install -y curl gnupg lsb-release
+sudo curl -sSL https://raw.githubusercontent.com/ros/rosdistro/master/ros.key  -o /usr/share/keyrings/ros-archive-keyring.gpg
+echo "deb [arch=$(dpkg --print-architecture) signed-by=/usr/share/keyrings/ros-archive-keyring.gpg] http://packages.ros.org/ros2/ubuntu $(lsb_release -cs) main" | sudo tee /etc/apt/sources.list.d/ros2.list > /dev/null
+```
+
+Update packages information and install ROS2 Galactic:
+```
+sudo apt update
+sudo apt install -y ros-galactic-desktop
+```
+
+### Get scenario simulator
+
+Install git
+```
+sudo apt install -y git
+```
+
+Create directory for a workspace and enter this directory.
+Throughout this document replace `<workspace_directory>` with a convenient name.
+```
+mkdir -p <workspace_directory>/src
+cd <workspace_directory>/src
+```
+
+Clone `scenario_simulator_v2` repository and enter it's directory:
+
+```
+git clone https://github.com/tier4/scenario_simulator_v2.git
+cd scenario_simulator_v2
+```
+
+### Downloading dependencies
+
+Install vcs tools
+```
+sudo apt install -y python3-vcstool
+```
+
+Download `scenario_simulator_v2` dependencies
+```
+vcs import scenario_simulator_v2/external < scenario_simulator_v2/dependency_galactic.repos
```

Comment:
After following this instruction and trying to execute this command after the last step from ### Get scenario simulator, I get 'no such file or directory' error. I suggest to remove line 84, as it's not needed.

Reply:
Good point, done